private-prediction/a - Secure Model Serving.ipynb

```diff
@@ -4,20 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>might have to be re-run to remove this error message</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dropoutlabs/tf-world-tutorial/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>